e2e/app/definition.go

```diff
@@ -238,12 +238,6 @@ func adaptNode(ctx context.Context, manifest types.Manifest, testnet *e2e.Testne
 		node.Seeds = append(node.Seeds, testnet.LookupNode(seed))
 	}
 
-	if node.Mode == types.ModeSeed {
```

Comment:
this resulted in staging seed node struggling to find peers. It even had a time where it didn't have any nodes. This is bad.